src/main/webapp/app/exercises/programming/assess/manual-result/programming-assessment-manual-result.service.ts

```diff
@@ -55,10 +55,11 @@ export class ProgrammingAssessmentManualResultService {
 
     /**
      * Deletes an assessment.
+     * @param participationId id of the participation, to which the assessment and the submission belong to
      * @param submissionId id of the submission, to which the assessment belongs to
      * @param resultId     id of the result which is deleted
      */
-    deleteAssessment(submissionId: number, resultId: number): Observable<void> {
-        return this.http.delete<void>(`${this.resourceUrl}/programming-submissions/${submissionId}/delete/${resultId}`);
+    deleteAssessment(participationId: number, submissionId: number, resultId: number): Observable<void> {
+        return this.http.delete<void>(`${this.resourceUrl}/participations/${participationId}/programming-submissions/${submissionId}/delete/${resultId}`);
```

Comment:
```suggestion
        return this.http.delete<void>(`${this.resourceUrl}/participations/${participationId}/programming-submissions/${submissionId}/results/${resultId}`);
```

Reply:
I don't see those changes reflected on the server and in the server integration tests (they fail at the moment)